lib/util/AbbreviationMappingsHelper.dart

```diff
@@ -0,0 +1,8 @@
+Map<String, String> position = {'RF' : 'Right Fielder', 'SS' : 'Short Stop', '1B' : 'First Baseman', '3B' : 'Third Baseman', 'CF' : 'Center Fielder', 'LF' : 'Left Fielder'};
```

Comment:
make this a more descriptive title to what type of map this is and maybe consider making this private if you would rather people use the function to get the full name

Reply:
Done